leku/src/main/java/com/adevinta/leku/LocationPickerActivity.kt

```diff
@@ -1213,11 +1216,13 @@ class LocationPickerActivity :
     }
 
     private fun setDefaultLocation() {
-        currentLocation = Location(getString(R.string.leku_network_resource))
-        currentLocation?.latitude = 0.0
-        currentLocation?.longitude = 0.0
-        setCurrentPositionLocation()
-        isLocationInformedFromBundle = true
+        if (ActivityCompat.checkSelfPermission(this, Manifest.permission.ACCESS_FINE_LOCATION) != PackageManager.PERMISSION_GRANTED) {
```

Comment:
I prefer you to use "PermissionUtils" class instead of calling for permissions directly. Example:

`if (PermissionUtils.isLocationPermissionGranted(applicationContext)) {`

Reply:
Thanks for the clarification! :D